recipes/r-rsymphony/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set version = '0.1-33' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rsymphony
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/Rsymphony_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/Rsymphony/Rsymphony_{{ version }}.tar.gz
+  sha256: 1ab18db7b5a52c284a21d7ec8d622640608ba6f7fd8d5b79cfc67e14ff6552f8
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - {{ stdlib("c") }}
+  host:
+    - r-base
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+
+test:
+  commands:
+    - $R -e "library('Rsymphony')"           # [not win]
+    - "\"%R%\" -e \"library('Rsymphony')\""  # [win]
+
+about:
+  home: https://R-Forge.R-project.org/projects/rsymphony, https://projects.coin-or.org/SYMPHONY,
+    https://www.coin-or.org/download/source/SYMPHONY/
+  license: EPL-1.0
```

Comment:
A copy of this license needs to be included. https://spdx.org/licenses/EPL-1.0.html